CHANGELOG.md

```diff
@@ -35,7 +35,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 - Cast Iron Frame, Sheet, and Tile
 - Sulfur Pools and Cast Iron features in Heat Lands
 - Wooden Lamp and Heat Lamp blocks, used in Wood and Heat lands respectively
-
+- Chocolatey Cake - the most scrumptious cake in the entire world!
```

Comment:
add a space between this line and ### Changed

Reply:
Done